### PR TITLE
Update aero params

### PR DESCRIPTION
**Pull request by nadiahouston**
- Aero params moved to easy_driver.py
- Vectorised aero math
- Better sideforce calculations

### CODE REVIEW

File: vehicle_params/easy_driver.py [hunk 1]
```diff
@@ -116,35 +116,72 @@ def __init__(self):
...
+        self.rad_to_deg = 180 / np.pi
+
+        # positions of component CoPs (magnitudes, equation takes signs into account)
+        # TODO: make positions relative to intermediate frame; these lines also seem wrong in general ATM
+        # Front, Undertray and Rear [x , y , z]
+        self.CoP = np.array([[23.65 * self.in_to_m + self.cg_total_position[0],  0, 9.30 * self.in_to_m],
+                             [-43.5 * self.in_to_m + self.cg_total_position[0],  0, 7.13 * self.in_to_m],
+                             [-67.6 * self.in_to_m + self.cg_total_position[0],  0, 42.91 * self.in_to_m]])
+
+        # gets aero coefficients for each component
```
Comment (Igor-BSouza) [inline vehicle_params/easy_driver.py:176]: Pretty sure we can delete this now

File: vehicle_params/easy_driver.py [hunk 2]
```diff
@@ -116,35 +116,72 @@ def __init__(self):
...
+        self.r_sens	=     [[-.018,  0,    0], [-.0137,  0,    0], [-.005,  -.0145, 0]]
+
+         # conversion factors
+        self.in_to_m = 0.0254
+        self.rad_to_deg = 180 / np.pi
+
+        # positions of component CoPs (magnitudes, equation takes signs into account)
+        # TODO: make positions relative to intermediate frame; these lines also seem wrong in general ATM
+        # Front, Undertray and Rear [x , y , z]
+        self.CoP = np.array([[23.65 * self.in_to_m + self.cg_total_position[0],  0, 9.30 * self.in_to_m],
```
Comment (Igor-BSouza) [inline vehicle_params/easy_driver.py:172]: Can we have this array be purely CoP location values from CAD measured in inches? So bring the calculation and CG offsets into the Aerodynamics file. I remember you started it during the workday but I may have gotten lost after all the vectorizing.

File: engine/aerodynamics.py [hunk 3]
```diff
@@ -12,82 +12,76 @@ def __init__(self, params:vehicle_params.BaseVehicle, logger:engine.Logger):
...
+                + np.cross(self.vehicle_params.CoP[1], part_force.T[1]) \
+                + np.cross(self.vehicle_params.CoP[2], part_force.T[2])
 
-            part_force = np.array([-Fd_part, Fs_part, -Fl_part])
-            forces = np.add(forces, part_force)
-            moments = np.add(moments, np.cross(self.CoP[i], part_force))
 
         # account for drag from rest of car
         drag_no_aero = 0.5 * self.__air_density * self.vehicle_params.CdA0 * x_dot ** 2
         forces[0] -= drag_no_aero
```
Comment (Igor-BSouza) [inline engine/aerodynamics.py:78]: @sarahfedorchak Can we make this and the other similar equation just one line?

File: vehicle_params/easy_driver.py (hunk 2 above)
Comment (nadiahouston) [reply vehicle_params/easy_driver.py]: Fixed in latest commit.

File: vehicle_params/easy_driver.py (hunk 1 above)
Comment (nadiahouston) [reply vehicle_params/easy_driver.py]: Fixed in latest commit.

File: engine/aerodynamics.py (hunk 3 above)
Comment (nadiahouston) [reply engine/aerodynamics.py:81]: I don't think that would be helpful because the equations aren't exactly the same.

Comment (Igor-BSouza) [reply engine/aerodynamics.py:82]: I mean the forces[0] and forces[1]

File: vehicle_params/easy_driver.py [hunk 4]
```diff
@@ -116,35 +116,59 @@ def __init__(self):
...
         self.CdA0 = 0.7155 # drag coefficient from non aero componenets
         self.static_ride_height = 0.0762 # m
+        self.CsA0 = 8.43
+
+        # distribution of downforce across components
+        self.ClA_dist = np.array([0.474, 0.289, 0.236])   # [front, undertray, rear]
+        self.CdA_dist = np.array([0.425, 0.178, 0.396])
+        self.CsA_dist = np.array([0.666, 0.000, 0.333])
+
+        # TODO: Update Sensitivities
```
Comment (Igor-BSouza) [inline vehicle_params/easy_driver.py:155]: Need to assign a new member to update these

**Review by kierancoz — APPROVED** [review]

**Pull request merged**